compiler/modules/nback/structure.bal

```diff
@@ -67,6 +67,78 @@ final RuntimeFunction mappingConstructFunction = {
 
 const LLVM_INDEX = "i32";
 
+// This is used in the name of the runtime function.
+enum ListReprKind {
+    LIST_GENERIC = "generic",
+    LIST_INT_ARRAY = "int_array",
+    LIST_FLOAT_ARRAY = "float_array"
+}
+
+type ListRepr readonly & object {
+    ListReprKind kind;
+    llvm:Type memberType;
+    int listDescGetIndex;
+    int listDescSetIndex;
```

Comment:
I could have introduced a separate object called `ListAccessorsRepr` and move `listDescGetIndex`, `listDescSetIndex` there. But I'd have to duplicate `buildMemberStore` as well.

Since (currently?) number of `ListRepr` happens to be same as number of accessors pairs, I can reuse `ListRepr`.